json-rpc/src/util.rs

```diff
@@ -207,3 +251,10 @@ pub fn script_view_from_script_function(script: &ScriptFunction) -> ScriptView {
         ..Default::default()
     }
 }
+
+pub fn sdk_language_from_user_agent(user_agent: &Option<String>) -> SdkLang {
```

Comment:
Change this function signature to take in `Option<&str>` its more idiomatic. Then if you have an `Option<String>` you'd call `as_ref()` on it before passing it into this function.